app/models/blog.py

```diff
@@ -12,15 +12,13 @@ class Blog(Base):
     __tablename__ = "blogs"
 
     id = Column(Integer, primary_key=True, index=True)
-    title = Column(String, index=True)
-    content = Column(String)
+    title = Column(String(2000), nullable=False, default="No title")
+    content = Column(String, nullable=True, default="")
     is_published = Column(Boolean, default=True)
-    tags = Column(String, default="")
+    tags = Column(String(255), default="")
```

Comment:
instead of string can we use any other type for tags? If in a post a user already added a multiple tags and now he just want to remove a specific tag from the tags list how FE can handle in that case.

Reply:
user can modify tags string. This also helps in searching using wildcards